From 826c4b3be0efd02d05db464c450c104104161b05 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 18 Nov 2021 17:06:21 +0800
Subject: [PATCH 56/64] Revert "drm/bridge: nwl-dsi: Avoid potential
 multiplication overflow on 32-bit"

This reverts commit be2e11b9f8fab49da85a3d79a34f4436c60510df.
---
 drivers/gpu/drm/bridge/nwl-dsi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/nwl-dsi.c b/drivers/gpu/drm/bridge/nwl-dsi.c
index 6cac2e58cd15..c65ca860712d 100644
--- a/drivers/gpu/drm/bridge/nwl-dsi.c
+++ b/drivers/gpu/drm/bridge/nwl-dsi.c
@@ -196,7 +196,7 @@ static u32 ps2bc(struct nwl_dsi *dsi, unsigned long long ps)
 	u32 bpp = mipi_dsi_pixel_format_to_bpp(dsi->format);
 
 	return DIV64_U64_ROUND_UP(ps * dsi->mode.clock * bpp,
-				  dsi->lanes * 8ULL * NSEC_PER_SEC);
+				  dsi->lanes * 8 * NSEC_PER_SEC);
 }
 
 /*
-- 
2.27.0

